spec/core/v2/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -596,15 +596,33 @@ function sendPacket(
     // increment the sequence. Thus there are monotonically increasing sequences for packet flow for a given clientId
     nextSequenceSend[sourceChannelId]=sequence+1
     
-    // log that a packet can be safely sent
-    // Event Emission 
-    emitLogEntry("sendPacket", {
+    // Event Emission for send packet
+    emitEvents("send_packet", {
       sourceId: sourceChannelId, 
-      destId: channel.counterpartyChannelId, 
-      sequence: sequence,
-      packet: packet,
-      timeoutTimestamp: timeoutTimestamp, 
+      destId: channel.counterpartyChannelId,
```

Comment:
I see these are defined as `bytes` in spec atm (for eth side of things where these are a contract address I assume?). Should they be hex encoded too in spec?

Reply:
Mmmm this is a good point. In ibc-go this is an alphanumeric string. It feels wasteful to convert that to a hex string only to convert it back. Tho it is possible that other implementations use bytes that do not decode to ASCII characters.

What do you think @gjermundgaraba ?